projects/jsoncpp/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+mkdir -p $WORK/jsoncpp
+cd $WORK/jsoncpp
+
+CFLAGS="$CFLAGS -DBORINGSSL_UNSAFE_FUZZER_MODE"
```

Comment:
Wait, what is this reference to boringssl? Was this code copied?
Same for next line.

Reply:
No use. Didn't need it. Forgot to remove it.